eks-distro-base/Makefile

```diff
@@ -746,6 +752,7 @@ test-all-minimal-images: $(addprefix test-minimal-images-, $(MINIMAL_VARIANTS))
 standard-update: UPDATE_TARGET=standard-images
 standard-update: IMAGE_NAME=eks-distro-base
 standard-update: IMAGE_COMPONENT=$(IMAGE_NAME)
+standard-update: SECURITY?=
```

Comment:
I dont *think* you want/need this here since this target is called as a part of the `update` target which will it accordingly.

Reply:
For some reason before this was added we were getting [test failures](https://prow.eks.amazonaws.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-distro-build-tooling/1139/eks-distro-base-test-presubmit/1735043820032102400)